buildpack.toml

```diff
@@ -17,301 +17,420 @@ api = "0.7"
+    purl = "pkg:generic/python@3.8.18?checksum=7c5df68bab1be81a52dea0cc2e2705ea00553b67107a301188383d7b57320b16&download_url=https://www.python.org/ftp/python/3.8.18/Python-3.8.18.tgz"
+    source = "https://www.python.org/ftp/python/3.8.18/Python-3.8.18.tgz"
+    source-checksum = "sha256:7c5df68bab1be81a52dea0cc2e2705ea00553b67107a301188383d7b57320b16"
     stacks = ["io.buildpacks.stacks.bionic"]
-    uri = "http://dash-app-resources.plotly-system.svc.cluster.local/packages/kpack-offline-files/python_3.7.15_linux_x64_bionic_d9bc98b9.tgz"
-    version = "3.7.15"
+    uri = "http://dash-app-resources.plotly-system.svc.cluster.local/packages/kpack-offline-files/python_3.8.18_linux_x64_bionic_bcc59a7f.tgz"
+    version = "3.8.18"
+
+  [[metadata.dependencies]]
+    checksum = "sha256:7c5df68bab1be81a52dea0cc2e2705ea00553b67107a301188383d7b57320b16"
+    cpe = "cpe:2.3:a:python:python:3.8.18:*:*:*:*:*:*:*"
+    id = "python"
+    licenses = ["0BSD", "CNRI-Python-GPL-Compatible", "PSF-2.0"]
+    name = "Python"
+    purl = "pkg:generic/python@3.8.18?checksum=7c5df68bab1be81a52dea0cc2e2705ea00553b67107a301188383d7b57320b16&download_url=https://www.python.org/ftp/python/3.8.18/Python-3.8.18.tgz"
+    source = "https://www.python.org/ftp/python/3.8.18/Python-3.8.18.tgz"
+    source-checksum = "sha256:7c5df68bab1be81a52dea0cc2e2705ea00553b67107a301188383d7b57320b16"
+    stacks = ["*"]
+    uri = "http://dash-app-resources.plotly-system.svc.cluster.local/packages/kpack-offline-files/Python-3.8.18.tgz"
+    version = "3.8.18"
 
   [[metadata.dependencies]]
-    checksum = "sha256:c9cecfe2e263d07ba681e419654e3a480f825702315480ec509117219106fb4c"
-    cpe = "cpe:2.3:a:python:python:3.7.16:*:*:*:*:*:*:*"
+    checksum = "sha256:6fbd433e406ad34f7482a5cca59dae593893d2e082811de9a54f6babeb8dd376"
+    cpe = "cpe:2.3:a:python:python:3.8.19:*:*:*:*:*:*:*"
     id = "python"
-    licenses = ["CNRI-Python-GPL-Compatible", "PSF-2.0"]
+    licenses = ["0BSD", "CNRI-Python-GPL-Compatible", "PSF-2.0"]
     name = "Python"
-    purl = "pkg:generic/python@3.7.16?checksum=0cf2da07fa464636755215415909e22eb1d058817af4824bc15af8390d05fb38&download_url=https://www.python.org/ftp/python/3.7.16/Python-3.7.16.tgz"
-    source = "https://www.python.org/ftp/python/3.7.16/Python-3.7.16.tgz"
-    source-checksum = "sha256:0cf2da07fa464636755215415909e22eb1d058817af4824bc15af8390d05fb38"
+    purl = "pkg:generic/python@3.8.19?checksum=c7fa55a36e5c7a19ec37d8f90f60a2197548908c9ac8b31e7c0dbffdd470eeac&download_url=https://www.python.org/ftp/python/3.8.19/Python-3.8.19.tgz"
+    source = "https://www.python.org/ftp/python/3.8.19/Python-3.8.19.tgz"
+    source-checksum = "sha256:c7fa55a36e5c7a19ec37d8f90f60a2197548908c9ac8b31e7c0dbffdd470eeac"
     stacks = ["io.buildpacks.stacks.jammy"]
-    uri = "http://dash-app-resources.plotly-system.svc.cluster.local/packages/kpack-offline-files/python_3.7.16_linux_x64_jammy_c9cecfe2.tgz"
```

Comment:
@tarzzz and/or @agateaux could you please confirm we no longer need to serve these files from `kpack-offline-files`?

Reply:
If you look well, as this diff is not easy to read, offline files are still there, it's just this old version which has been replaced by a newer one like `kpack-offline-files/python_3.8.18_linux_x64_jammy_58676489`.

Python 3.7 has been removed from the upstream.

Reply:
Hm, sorry, I think I'm still missing something : ) I see _some_ references to `kpack-offline-files` still being preserved but I see a lot of changes here that also shows the python package versions being served from `/packages/` and not `/packages/kpack-offline-files`. Is this intended? 

For example: https://github.com/plotly/paketo-buildpacks_cpython/pull/11/files#diff-5f54cb49d0e28117084fad482c0daf89502d4ecb329230071fcf78e56dc939b3R68

Reply:
Ok, this one I need a review. It was added on the upstream, so I also put it there.
Should we remove this type?